client/reader/stream/index.jsx

```diff
@@ -164,7 +164,7 @@ export default class ReaderStream extends React.Component {
 			recs,
 			updateCount: store.getUpdateCount(),
 			selectedIndex: store.getSelectedIndex(),
-			isFetchingNextPage: store.isFetchingNextPage(),
+			isFetchingNextPage: store.isFetchingNextPage && store.isFetchingNextPage(),
```

Comment:
I didn't understand what was going on, but I figured it would be best to patch the hole and debug later